File: DiarioDigital-Front-End-main/src/app/DTO/response/capitulo/CapituloResponse.ts
export class CapituloResponse {
  id: number;
  titulo: string;
  Conteudo:string;
  numeroCapitulo: number;

  constructor(id: number, titulo: string,Conteudo:string, numeroCapitulo: number) {
    this.id = id;
    this.titulo = titulo;
    this.Conteudo = Conteudo;
    this.numeroCapitulo = numeroCapitulo;
  }
}

File: DiarioDIgital-main/src/main/java/com/LSoftwareLTDA/diarioDigital/controller/dto/capitulos/CapituloResponse.java
package com.LSoftwareLTDA.diarioDigital.controller.dto.capitulos;

import com.LSoftwareLTDA.diarioDigital.entidades.Capitulo;

public record CapituloResponse(Long id, String titulo, String Conteudo, int numeroCapitulo) {

	public CapituloResponse(Capitulo entidade) {
		this(entidade.getId(), entidade.getTitulo(), entidade.getConteudo(), entidade.getNumeroCapitulo());
	}
}
